config/install/field.field.node.species.field_hair_colors.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_hair_colors
+    - node.type.species
+id: node.species.field_hair_colors
+field_name: field_hair_colors
+entity_type: node
+bundle: species
+label: 'Hair Colors'
```

Comment:
This field label uses title case but others on the Species content type use sentence case. However you want to fix, it would just be good to see consistency.
![species_field_hair_colors_label](https://github.com/notS0Solo/swapimigration/assets/1434979/0cb8f59a-e602-4855-8c93-c06f68e757da)

Reply:
Fixed